### PR TITLE
Certora - Continuous Formal Verification Nov 22 Proposal

### PR DESCRIPTION
**Pull request by yuradmt**

### CODE REVIEW

File: content/aips/CERTORA-CONTINUOUS-FORMAL-VERIFICATION.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Certora Continuous Formal Verification
+author: Shelly Grossman (@shellygr)
+shortDescription: A continuation of proposal 6308 to improve the security of smart contracts built on top of Aave using a combination of formal verification and manual code review.
```
Comment (eboadom) [inline content/aips/CERTORA-CONTINUOUS-FORMAL-VERIFICATION.md:4]: Better to refer if needed to the proposal id on the governance of the previous engagement. Not sure what is the `6308`

**Comment by eboadom** [comment]
I would probably add the `2022-23` suffix to the filename

File: content/aips/CERTORA-CONTINUOUS-FORMAL-VERIFICATION.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+title: Certora Continuous Formal Verification
+author: Shelly Grossman (@shellygr)
+shortDescription: A continuation of proposal 6308 to improve the security of smart contracts built on top of Aave using a combination of formal verification and manual code review. 
+discussions: https://governance.aave.com/t/security-and-agility-of-aave-smart-contracts-via-continuous-formal-verification/10181
+created: 2022-11-08
+
```
Comment (sakulstra) [inline content/aips/CERTORA-CONTINUOUS-FORMAL-VERIFICATION.md:7]:
```suggestion
```

File: content/aips/CERTORA-CONTINUOUS-FORMAL-VERIFICATION.md (hunk 1 above)
Comment (yuradmt) [reply content/aips/CERTORA-CONTINUOUS-FORMAL-VERIFICATION.md]: Changed to proposal 66 (https://app.aave.com/governance/proposal/66/)

**Comment by yuradmt** [comment]
Committed fixes per the remarks

**Pull request merged**